pytorch_toolkit/ote/ote/modules/exporters/mmdetection.py

```diff
@@ -65,9 +87,48 @@ def _export_to_openvino(self, args, tools_dir):
                               f'{args["config"]} '
                               f'{args["load_weights"]} '
                               f'{os.path.join(args["save_model_to"], "alt_ssd_export")} '
+                              f'{update_config} '
+                              f'--opset={self.opset} '
                               f'openvino '
                               f'--input_format {args["openvino_input_format"]} '
                               f'--alt_ssd_export ')
 
     def _get_tools_dir(self):
         return MMDETECTION_TOOLS
+
+
+@EXPORTERS.register_module()
+class MMDetectionCustomClassesExporter(MMDetectionExporter):
+
+    def _get_update_config(self, args):
+        classes_from_snapshot = load_classes_from_snapshot(args['load_weights'])
+        if not classes_from_snapshot:
+            raise RuntimeError(f'There are no CLASSES in meta information of snapshot: {args["load_weights"]}')
+
+        if 'classes' in args and args['classes']:
+            classes_from_args = args['classes'].split(',')
+            if classes_from_args != classes_from_snapshot:
+                raise RuntimeError('Set of classes passed through CLI does not equal to classes stored in snapshot: '
+                                   f'{classes_from_args} vs {classes_from_snapshot}')
+
+        update_config_dict = classes_list_to_update_config_dict(args['config'], classes_from_snapshot)
+        update_config = '--update_config ' + ' '.join(f'{k}={v}' for k,v in update_config_dict.items())
+        update_config = update_config.replace('"', '\\"')
+
+        return update_config
+
+    def _dump_classes(self, args):
+        classes = load_classes_from_snapshot(args['load_weights'])
+        out_folder = args['save_model_to']
+        out_basename = os.path.splitext(args['config'])[0] + '.extra_params'
+        import yaml
```

Comment:
Could you move this import yaml to the header of the file? (it is safe)

Reply:
fixed

Reply:
fixed